src/components/DeeplinkWrapper/deeplinkRoutes.js

```diff
@@ -21,6 +22,9 @@ export default [
     {
         // /v/*
         pattern: '/v($|(//*))',
+        // It was decided in the team to disable deep linking in desktop App when passwordless is enabled.
+        // We want to open the magic link into its own tab
```

Comment:
```suggestion

        // Disable deep linking in desktop App when passwordless is enabled because
        // we want to open the magic link in its own tab
```